MobileCenterPush/MobileCenterPush/Support/MobileCenterPush.xcconfig

```diff
@@ -0,0 +1,3 @@
+#include "../../../Global.xcconfig"
```

Comment:
You may also force the non support for app extensions here as they can't use pushes anyway only their containing app can. This will also remove the Xcode errors about restricted APIs while using `sharedApplication`.